Training/view/frontend/templates/account/dashboard/credithold.phtml

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+/**
+ * @var Codifi\Training\Block\Account\Dashboard\CreditHold $block
+ */
+?>
+<?php if (
+    $block->getOptionCreditHoldEnable() === 1 &&
+    $block->getCustomerAttr() === 1 &&
+    $block->getFlag() !== true
+): ?>
+    <div class="confirmation-modal-content">
+        <p><?= $block->getMessage() ?></p>
+    </div>
+    <script>
+        require([
+            'jquery',
+            'Magento_Ui/js/modal/confirm'
+        ], function ($) {
+            'use strict';
+
+            $('.confirmation-modal-content').confirm({
+                actions: {
+                    confirm: function() {
```

Comment:
Review "Magento_Ui/js/modal/confirm" and make sure that this empty methods should present here